src/utils/popups.js

```diff
@@ -27,10 +27,18 @@ const toTableCells = (displayProps) => {
 };
 
 const basicPopupTableHTML = (title, displayProps) => {
-  return `<table>
-       <caption style="font-weight: bold; caption-side: top">${title}</caption>
+  let scrollCss = "";
+  if (Object.keys(displayProps).length > 5) {
+    scrollCss = `style="height: 20rem; overflow: auto; scrollbar-width: thin; padding-bottom: 5px;"`;
+  }
```

Comment:
Not sure if this is the nicest, but making the popup table scrollable for popups with more rows - basically just for Bristol really

Reply:
This is great! will also be useful for the changes I made that show several nearby watercourse links in the popup..